robottelo/decorators/host.py

```diff
@@ -1,56 +1,36 @@
+"""Implements decorator regarding satellite host"""
 import logging
 import re
-import six
 import unittest2
 
+
```

Comment:
In order to follow Robottelo import style, there should be two blocks, first with the import statements then a blank line and them the from ... import statements. Each block should be sorted alphabetically